boxoffice/mailclient.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from flask import render_template
+from flask.ext.mail import Message
+from html2text import html2text
+from premailer import transform as email_transform
+from boxoffice.models import Order
+from boxoffice import mail, app
```

Comment:
For coding style I'd prefer these to be `from ..models import …` and `from .. import …`